Assets/Scripts/Health/LivesUI.cs

```diff
@@ -32,10 +32,16 @@ private void Start()
 				rTransform.position = new Vector2(rTransform.rect.width + (size / 4 + rTransform.rect.width) * i, Screen.height - size);
 				hearts.Add(life);
 			}
-			lives.LifeLostEvent += OnLiveLostEvent;
+			lives.LifeLostEvent += OnLifeLostEvent;
+			lives.AllLifeLost += OnAllLifeLostEvent;
 		}
 
-		private void OnLiveLostEvent(int obj)
+		private void OnAllLifeLostEvent()
+		{
+			lives.LifeLostEvent -= OnLifeLostEvent;
+		}
+
+		private void OnLifeLostEvent(int obj)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'obj'.](https://app.codacy.com/manual/stepperman/pvb-programming/pullRequest?prid=5401570)